### PR TITLE
Implement seed service with commands to run seeds

### PR DESCRIPTION
**Pull request by jerben**

### CODE REVIEW

File: src/sihl/sihl.ml [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-module Log = Log
-module Data = Data
-module Web = Web
-module Cmd = Cmd
-module Schedule = Schedule
-module Authz = Authz
 module App = App
-module Token = Token
 module Authn = Authn
+module Authz = Authz
```
Comment (jerben) [inline src/sihl/sihl.ml:3]: Sort modules alphabetically

File: src/sihl/sihl.mli [hunk 2]
```diff
@@ -34,74 +34,78 @@ let _ = App.(empty |> with_services services |> with_routes routes |> run)
```
Comment (jerben) [inline src/sihl/sihl.mli:34]: Sort modules alphabetically

File: src/seed/seed.mli [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+A disadvantage of this approach is, that seed data generation requires a Sihl app with services. *)
+
+type t = Seed_core.t
+
+(** {1 Installation}
+
+{[
+module Log = Sihl.Log.Service.Make ()
+module Cmd = Sihl.Cmd.Service.Make ()
+module Config = Sihl.Cmd.Service.Make (Config)
```
Comment (aronerben) [inline src/seed/seed.mli:19]: Why is this called `Config`? Why is it needed?

File: src/seed/service.ml [hunk 4]
```diff
@@ -0,0 +1,66 @@
+open Base
+module Sig = Sig
+
+let registered_seeds = ref (Map.empty (module String))
```
Comment (aronerben) [inline src/seed/service.ml:4]: Why are you using mutable state?

File: src/seed/seed.mli (hunk 3 above)
Comment (jerben) [reply src/seed/seed.mli]: Fixed.

File: src/seed/service.ml (hunk 4 above)
Comment (jerben) [reply src/seed/service.ml:4]: Because I want to be able to register seeds after the service was started. I don't see how to avoid mutable state here.

**Pull request merged**